LLMonFHIR/Settings/Prompt.swift

```diff
@@ -45,7 +45,10 @@ enum Prompt: String {
     
     
     var prompt: String {
-        UserDefaults.standard.string(forKey: rawValue) ?? defaultPrompt
+        let preferredLanguage = Locale.preferredLanguages[0]
+        var prompt = UserDefaults.standard.string(forKey: rawValue) ?? defaultPrompt
+        prompt += "\n Chat with the user in \(preferredLanguage)"
```

Comment:
We should ideally also localize this string, just to be consistent with the rest of the application, we also localize all prompts.